index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <script
+      src="https://kit.fontawesome.com/f0e9415ee2.js"
```

Comment:
- [ ] Your styling is not having an effect on your page. please connect your styles to your HTML correctly so that your page can look similar to the one from Figma design. 
![image](https://user-images.githubusercontent.com/43877689/176633915-699aed69-fb48-4fdd-8021-3b32778a3ab7.png)

Reply:
@ishimwezachee I thought the styling was mobile-first i.e. I set a media query to max-width 480 px which is why the CSS was not working. I realize that now, but I've changed it. Thanks